tests/specs/scoped.js

```diff
@@ -592,7 +592,7 @@ test('scoped Store .off()', function (t) {
   })
 })
 
-test('when type change', function (t) {
+test.only('when type change', function (t) {
```

Comment:
Makes sense, depending on how badly I wanted more rules I would create my own preset/config (which could include standard) and use that in all repos (so it would be similar to standard, except your own stuff too)